Enable the necessary options for Ethernet on the mb618 by default.

Signed-off-by: Stuart Menefy <stuart.menefy@st.com>
Signed-off-by: Giuseppe Cavallaro <peppe.cavallaro@st.com>

--- linux-2.6.23-stm.orig/arch/sh/configs/mb618_defconfig	2008-03-18 07:16:25.000000000 +0100
+++ linux-2.6.23-stm/arch/sh/configs/mb618_defconfig	2008-03-18 07:17:18.000000000 +0100
@@ -522,7 +522,7 @@ CONFIG_NETDEVICES=y
 # CONFIG_MACVLAN is not set
 # CONFIG_EQUALIZER is not set
 # CONFIG_TUN is not set
-# CONFIG_NWHW_CONFIG is not set
+CONFIG_NWHW_CONFIG=y
 CONFIG_PHYLIB=y
 
 #
@@ -534,11 +534,10 @@ CONFIG_PHYLIB=y
 # CONFIG_LXT_PHY is not set
 # CONFIG_CICADA_PHY is not set
 # CONFIG_VITESSE_PHY is not set
-# CONFIG_SMSC_PHY is not set
+CONFIG_SMSC_PHY=y
 # CONFIG_BROADCOM_PHY is not set
 # CONFIG_ICPLUS_PHY is not set
 # CONFIG_STE10XP is not set
-# CONFIG_LAN8700 is not set
 # CONFIG_FIXED_PHY is not set
 CONFIG_NET_ETHERNET=y
 CONFIG_MII=y
@@ -546,7 +545,7 @@ CONFIG_MII=y
 # CONFIG_SMSC911x is not set
 # CONFIG_SMC91X is not set
 CONFIG_NETDEV_1000=y
-# CONFIG_STMMAC_ETH is not set
+CONFIG_STMMAC_ETH=y
 CONFIG_NETDEV_10000=y
 
 #
--- linux-2.6.23-stm.orig/arch/sh/configs/mb618se_defconfig	2008-03-18 07:16:38.000000000 +0100
+++ linux-2.6.23-stm/arch/sh/configs/mb618se_defconfig	2008-03-18 07:17:26.000000000 +0100
@@ -522,7 +522,7 @@ CONFIG_NETDEVICES=y
 # CONFIG_MACVLAN is not set
 # CONFIG_EQUALIZER is not set
 # CONFIG_TUN is not set
-# CONFIG_NWHW_CONFIG is not set
+CONFIG_NWHW_CONFIG=y
 CONFIG_PHYLIB=y
 
 #
@@ -534,11 +534,10 @@ CONFIG_PHYLIB=y
 # CONFIG_LXT_PHY is not set
 # CONFIG_CICADA_PHY is not set
 # CONFIG_VITESSE_PHY is not set
-# CONFIG_SMSC_PHY is not set
+CONFIG_SMSC_PHY=y
 # CONFIG_BROADCOM_PHY is not set
 # CONFIG_ICPLUS_PHY is not set
 # CONFIG_STE10XP is not set
-# CONFIG_LAN8700 is not set
 # CONFIG_FIXED_PHY is not set
 CONFIG_NET_ETHERNET=y
 CONFIG_MII=y
@@ -546,7 +545,7 @@ CONFIG_MII=y
 # CONFIG_SMSC911x is not set
 # CONFIG_SMC91X is not set
 CONFIG_NETDEV_1000=y
-# CONFIG_STMMAC_ETH is not set
+CONFIG_STMMAC_ETH=y
 CONFIG_NETDEV_10000=y
 
 #
